diff -ru openvas-manager-7.0.1/CMakeLists.txt openvas-manager-7.0.1.new/CMakeLists.txt
--- openvas-manager-7.0.1/CMakeLists.txt	2017-02-01 11:17:47.000000000 +0100
+++ openvas-manager-7.0.1.new/CMakeLists.txt	2017-04-23 23:01:48.820225149 +0200
@@ -187,13 +187,14 @@
 set (OPENVAS_CERT_RES_DIR "${OPENVAS_DATA_DIR}/cert")
 set (OPENVASMD_STATE_DIR  "${OPENVAS_STATE_DIR}/openvasmd")
 set (OPENVAS_CA_DIR       "${OPENVASMD_STATE_DIR}/trusted_certs")
+set (OPENVAS_CERT_DIR     "${SYSCONFDIR}/pki/openvas")
 set (OPENVAS_LIB_INSTALL_DIR     "${LIBDIR}")
 
-set (OPENVAS_SCANNER_CERTIFICATE "${OPENVAS_STATE_DIR}/CA/servercert.pem")
-set (OPENVAS_SCANNER_KEY         "${OPENVAS_STATE_DIR}/private/CA/serverkey.pem")
-set (OPENVAS_CLIENT_CERTIFICATE  "${OPENVAS_STATE_DIR}/CA/clientcert.pem")
-set (OPENVAS_CLIENT_KEY          "${OPENVAS_STATE_DIR}/private/CA/clientkey.pem")
-set (OPENVAS_CA_CERTIFICATE      "${OPENVAS_STATE_DIR}/CA/cacert.pem")
+set (OPENVAS_SCANNER_CERTIFICATE "${OPENVAS_CERT_DIR}/CA/servercert.pem")
+set (OPENVAS_SCANNER_KEY         "${OPENVAS_CERT_DIR}/private/CA/serverkey.pem")
+set (OPENVAS_CLIENT_CERTIFICATE  "${OPENVAS_CERT_DIR}/CA/clientcert.pem")
+set (OPENVAS_CLIENT_KEY          "${OPENVAS_CERT_DIR}/private/CA/clientkey.pem")
+set (OPENVAS_CA_CERTIFICATE      "${OPENVAS_CERT_DIR}/CA/cacert.pem")
 
 if (NOT OPENVAS_RUN_DIR)
   set (OPENVAS_RUN_DIR      "${LOCALSTATEDIR}/run")
